keepassxc-browser/background/keepass.js

```diff
@@ -804,7 +804,7 @@ keepass.disableAutomaticReconnect = function() {
     keepass.reconnectLoop = null;
 };
 
-keepass.reconnect = async function(tab = null, connectionTimeout = 1500) {
+keepass.reconnect = async function(tab = null, connectionTimeout = page.settings.connectionTimeout) {
     keepassClient.connectToNative();
     keepass.generateNewKeyPair();
     const keyChangeResult = await keepass.changePublicKeys(tab, !!connectionTimeout, connectionTimeout).catch(() => false);
```

Comment:
Need to do another Min/Max check here just to be sure passed values make sense and won't cause undesirable behavior.